downloads/README.md

```diff
@@ -207,7 +225,7 @@ You can also try the script [hello world](http://opendataplane.org/hello-world/)
 
 ### OpenDataPlane odp-linux vs odp-dpdk
 
-**odp-linux** can interface with DPDK poll mode drivers as an option vs the socket interface. However it retains everything else that is part of the generic Linux implementation, including for example the buffer management implementation. Therefore it has to copy everything during receive and transmit between its own buffers and DPDK ones.  
+**odp-linux** can interface with DPDK poll mode drivers as an option vs the socket interface. However it retains everything else that is part of the generic Linux implementation, including for example the buffer management implementation. Therefore it has to copy everything during receive and transmit between its own buffers and DPDK ones.
```

Comment:
odp-linux actually support zero-copy now. So, you can remove the last sentence.

Also, remove also "odp-nextmap" chapter under. We do not need to mention what implementations do not exist.